test/comparison-tests/customTransformer/expectedOutput-2.3/output.txt

```diff
@@ -0,0 +1,5 @@
+    Asset     Size  Chunks             Chunk Names
+bundle.js  2.92 kB       0  [emitted]  main
+chunk    {0} bundle.js (main) 193 bytes [entry] [rendered]
+    [0] ./.test/customTransformer/SUBMODULE/SUBMODULE.ts 78 bytes {0} [built]
```

Comment:
In actual fact that is the reason the test is failing on Travis right now I think. Case insensitive Windows Vs case sensitive Linux :smile: I'll change the test so it is 1 file only (no imports) tonight and then we should have a reliable test that works cross platform. Then I think we're ready to merge!